src/pages/index.js

```diff
@@ -95,6 +99,10 @@ const getServerSideProps = async (context) => {
 
     const { isSuccessfulSearch, searchResponseData } = await searchForCourses(searchTerm);
 
+    if (!isSuccessfulSearch) {
+        console.log(logEntry(context.req, LOG_TYPES.ERROR, context.query, { results: searchResponseData.results }));
```

Comment:
:slightly_smiling_face: 

```suggestion
        console.error(logEntry(context.req, LOG_TYPES.ERROR, context.query, { results: searchResponseData.results }));
```